pkg/ingestor/parser/dsse/parser_dsse_test.go

```diff
@@ -17,21 +17,28 @@ package dsse
 
 import (
 	"context"
+	"sync"
 	"testing"
 
-	"github.com/google/go-cmp/cmp"
 	"github.com/guacsec/guac/internal/testing/mockverifier"
+	"github.com/guacsec/guac/pkg/ingestor/verifier"
+
+	"github.com/google/go-cmp/cmp"
 	"github.com/guacsec/guac/internal/testing/testdata"
 	"github.com/guacsec/guac/pkg/assembler"
 	"github.com/guacsec/guac/pkg/handler/processor"
 	"github.com/guacsec/guac/pkg/ingestor/parser/common"
-	"github.com/guacsec/guac/pkg/ingestor/verifier"
 	"github.com/guacsec/guac/pkg/logging"
 )
 
+var once sync.Once
```

Comment:
see comment: https://github.com/guacsec/guac/pull/1345/files#r1346104106. Previous PR might also need to be fixed.

Reply:
A friendly ping @pxp928 @lumjjb pertaining https://github.com/guacsec/guac/pull/1345#discussion_r1346213382